### PR TITLE
Update Report Abuse Page

### PR DESCRIPTION
**Pull request by ryuyu**
This PR updates the report abuse page with the following:
1. Hides the block that only pertains to copyright claims when the report reason is not copyrighting.
2. Moves validation of Signature to controller from model.
3. Updates option to read "copyright" instead of "license"
4. Adds a small blurb about contact via an email address.

See https://github.com/NuGet/NuGetGallery/issues/3792 for more information.

@karann-msft @xavierdecoster @skofman1

### CODE REVIEW

**Comment by skofman1** [comment]
screen shot?

**Comment by ryuyu** [comment]
Not copyright claim:
image

Copyright claim:
image

File: src/NuGetGallery/ViewModels/ReportPackageReason.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ public enum ReportPackageReason
         [Description("The package contains malicious code")]
         ContainsMaliciousCode,
 
-        [Description("The package violates a license I own")]
+        [Description("The package violates a copyright I own")]
         ViolatesALicenseIOwn,
```
Comment (loic-sharma) [inline src/NuGetGallery/ViewModels/ReportPackageReason.cs]: Should the `ViolatesALicenseIOwn` also be updated to something like `ViolatesACopyrightIOwn`?

File: src/NuGetGallery/Views/Packages/ReportAbuse.cshtml [hunk 2]
```diff
@@ -88,6 +88,7 @@
             <span class="field-hint-message">Sign with your real name.</span>
             @Html.ValidationMessageFor(m => m.Signature)
         </div>
+        <div class="infringement-claim-requirements">You can also send us an email at <a href="mailto:admn@nuget.org">admin@nuget.org</a></div>
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/ReportAbuse.cshtml]: Is this email right, or should it be `admin@nuget.org`?

Comment (ryuyu) [inline src/NuGetGallery/Views/Packages/ReportAbuse.cshtml]: yes.

File: src/NuGetGallery/ViewModels/ReportPackageReason.cs (hunk 1 above)
Comment (ryuyu) [inline src/NuGetGallery/ViewModels/ReportPackageReason.cs]: We'll just change the visible description for now.

**Pull request merged**